### ISSUE TITLE
Custom column headers

### ISSUE THREAD

**Issue by strengejacke** [r-lib/pillar#63]
Just as reminder related to your suggestion here: 
https://github.com/strengejacke/sjlabelled/issues/3#issuecomment-341638453

**Comment by krlmlr** [comment]
Thanks. We'll need to think about a `pillar_capital()` entry point, but only after first CRAN release.

**Comment by krlmlr** [comment]
Closing in favor of #73.
